actors/verifreg/src/lib.rs

```diff
@@ -559,13 +594,26 @@ impl Actor {
                         continue;
                     }
 
+                    let prev_max = claim.term_max;
```

Comment:
Just inline this

Reply:
We're not adding previous values anymore.